_blog-src/_layouts/post.html

```diff
@@ -11,6 +11,21 @@ <h1 style="color: #666; font-weight: bold" class="post-title section-header">{{
     <span style="color: #243446; margin-left: 15px; font-size: 13px"><i class="fa fa-tags"></i> {{ page.categories | array_to_sentence_string }}</span>
   </header>
 
+
+  {% if page.newUrl || page.oldUrl %}
+  <div class="name-change-alert" style="margin-top:15px;">
+    {% if page.newUrl %}
+    <!-- Old Post -->
+    <p>NOTE: A newer revision of this blogpost correct for <a href="{{ site.url }}{{ site.baseurl }}/{{ page.newUrl }}">apiman 1.2.x is available</a>. This version written for apiman 1.1.x, for which it remains valid.</p>
+    {% endif %}
+
+    {% if page.oldUrl %}
+    <!-- New Post -->
+    <p>NOTE: This is a redux of our popular Keycloak OAuth2 for *apiman 1.2.x*. If you're still using apiman 1.1.x, you can refer to the <a href="{{ site.url }}{{ site.baseurl }}/{{ page.oldUrl }}">older revision</a>.</p>
```

Comment:
> NOTE: This is a redux of our popular Keycloak OAuth2 

I guess this is artefact when you copied over the 'old/new blog post' notices from the KC blogpost. Presume you'll want to make it more generic. In asciidoctor `NOTE:` is an admonition, and it doesn't work in markdown, so suggest you kill that bit.